extension-manifest-v3/src/pages/panel/views/home.js

```diff
@@ -93,6 +93,31 @@ function setStatsType(host, event) {
   store.set(host.options, { panel: { statsType: type } });
 }
 
+const trackerStatus = ({ count, title, description, icon, iconColor }) => html`
```

Comment:
This looks to me like a visual component, so please make a new one in `/ui/src/modules/panel/componens` (We keep them there for the possible sharing of the UI with MV2, but I think it might never happen, so eventually we can move it to he MV3 source)

Reply:
why encapsulate something that is not shared?

Reply:
There are a few separate things:

1. It should be a component, as it is then easier to debug (the function just disappears, when called). Try to avoid creating a nested render function. It then is obvious that it should be a component (with possible slots, so you don't have to pass all of those arguments).
2. Moving it to the `ui` - this is what I explained in the brackets - it could be in the `src/pages/panel/components` as well, but for now we keep panel components in the `ui` module. And there is another rule - one file for one component, flat structure, with auto-definition, so you can just create a file there with a new component.